### PR TITLE
[batch] Update environment.yml and build image

### PR DESCRIPTION
**Pull request by jigold**
Making one change with all of the SQL packages needed so I can speed up testing of other branches.

### CODE REVIEW

File: batch/environment.yml [hunk 1]
```diff
@@ -11,3 +11,13 @@ dependencies:
   - pylint
   - pytest
   - requests
+  - aiohttp
+  - aiodns
```
Comment (akotlar) [inline batch/environment.yml]: We’re doing dns resolution?

Comment (akotlar) [reply batch/environment.yml]: I didn’t see that 
https://github.com/hail-is/hail/pull/5563/files or the cloud MySQL pr’s but I may have missed

File: batch/environment.yml [hunk 2]
```diff
@@ -11,3 +11,13 @@ dependencies:
...
+  - cchardet
+  - aiohttp_jinja2
+  - jinja2
+  - uvloop>=0.12
```
Comment (akotlar) [inline batch/environment.yml]: Do we need the version resolution? The current package is 0.12.1 right? I’m actually not sure when > resolution is needed, unless we can’t rely on the package repo to point version to latest?

Comment (jigold) [reply batch/environment.yml]: See here: 
https://github.com/MagicStack/uvloop#using-uvloop

Syntax for using uvloop changed with that version.

File: batch/environment.yml (hunk 1 above)
Comment (jigold) [reply batch/environment.yml]: It's recommended to install it for aiohttp, so I added it: 
https://aiohttp.readthedocs.io/en/stable/#library-installation

**Comment by jigold** [comment]
I need to fix the pipeline pylint errors in this branch as well...

File: batch/environment.yml (hunk 2 above)
Comment (akotlar) [reply batch/environment.yml]: Right, I understand why you want >= 0.12, I just meant that by default, if we write 
`
- uvloop
` 

(no version) are there circumstances in which a version < current will be fetched? I'm more used to constraining versions to be something smaller than current, such as ^0.12.3 or <0.19.

File: batch/environment.yml (hunk 1 above)
Comment (akotlar) [reply batch/environment.yml]: True, nice catch. I'm still looking where it's used; do you know? I'm still digging through the source code. I see it's used in `resolve.py` and `TCPConnector`, but still need to figure out how those are used.

I'll approve, and open an issue to track that.

**Comment by jigold** [comment]
@cseed Is the timeline for removing conda throughout the project this week? If so, then I'll close this PR since those changes will remove the need for my changes to the conda environment.

**Pull request closed without merging**